### ISSUE TITLE
BluePrint : add support for indexes

### ISSUE THREAD

**Issue by Fen747** [bluelibs/bluelibs#174]
A very nice to have feature would be to enable indexes to be added from `Blueprint` config files, like so :

```typescript
Studio.field.string({
})
```

**Comment by theodorDiaconu** [comment]
Agreed. This is also part of Blueprint v2, where we put everything into `base` folder and properly merge class interfaces with GraphQL entities.
